### PR TITLE
Added feature works/about section to the portfolio

### PR DESCRIPTION
**Pull request by r-ahmed2022**
In this pull request, I added the middle section of the website:-
1.  Added "Works" section
2. Added  "About Myself" section

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -195,6 +192,394 @@ body {
...
+  width: 124px;
+  height: 40px;
+  margin-left: 5px;
+  order: 2;
+  color: #fff;
+  flex: none;
+  cursor: pointer;
+  border: 0;
+}
+
```
Comment (tresorsawasawa) [inline css/style.css:317]: - [ ] Make sure your button follows the button styling:  **Hover** and **Pressed** check all correct properties assigned as for **color** for Figma Design.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @r-ahmed2022 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Required Changes :recycle::recycle: 
- [ ] I think it would be better if you remove the **overflow-auto**, especially on the **about section** and the and the **overflow-x** at the entire page. It makes you don't fix correctly project's elements at their right position.
- [ ] After you remove the overflow, make sure your design matches perfectly the one from figma by respecting the **spacing**, **colors**, and **backgrounds positions** of your tool in the page. 
- [ ] **Check other comments under the review**.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## NOTICE
- [ ] I only reviewed the project on the requested part as the student went beyond the requested milestone.

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -110,6 +110,183 @@ <h1 class="name-text">Hey There. <br>
...
+          <div id="works-title">
+           <p class="works-title-text">My Recent Works</p>
+           <img src="images/line.svg" id="works-title-img"  alt="my works profile">
+          </div>
+          <div class="card-1">
+                <div id="div-1--left">
+                <img id="card-1-img" src="images/recenttitle.svg" alt="recent works">
+                </div>
+                <div id="div-1--right">
+                <p class="works-title-text" id="works-title-text-1">Multi-Post Stories</p>
```
[the remark this answers is not in the document]
Comment (r-ahmed2022) [reply index.html:124]: Done as advised

File: index.html [hunk 3]
```diff
@@ -110,6 +110,183 @@ <h1 class="name-text">Hey There. <br>
...
+                  <p class="card-B-para">
+                    A daily selection of privately personalized <br> 
+                    reads;no accounts or sign-ups required.<br>
+                     has been the industry's standard
+                   </p>
+                   <div class="card-B-btns">
+                   <button type="button" class="btns-3">html</button>
+                   <button type="button" class="btns-3">bootstrap</button>
+                   <button type="button" class="btns-3">Ruby</button>
+                   </div>
```
[the remark this answers is not in the document]
Comment (r-ahmed2022) [reply index.html:158]: done as advised

File: index.html [hunk 4]
```diff
@@ -110,6 +110,183 @@ <h1 class="name-text">Hey There. <br>
...
```
[the remark this answers is not in the document]
Comment (r-ahmed2022) [reply index.html:174]: Done as advised

File: index.html [hunk 5]
```diff
@@ -110,6 +110,183 @@ <h1 class="name-text">Hey There. <br>
...
+                    <p class="card-B-para">
+                      A daily selection of privately personalized <br> 
+                      reads;no accounts or sign-ups required.<br>
+                       has been the industry's standard
+                     </p>
+                     <div class="card-B-btns">
+                     <button type="button" class="btns-3">html</button>
+                     <button type="button" class="btns-3">bootstrap</button>
+                     <button type="button" class="btns-3">Ruby</button>
+                     </div>
```
[the remark this answers is not in the document]
Comment (r-ahmed2022) [reply index.html:208]: Done as advised

File: index.html [hunk 6]
```diff
@@ -110,6 +110,183 @@ <h1 class="name-text">Hey There. <br>
...
+                </div> 
+            </div>
```
[the remark this answers is not in the document]
Comment (r-ahmed2022) [reply index.html:226]: Done as advised

File: css/style.css (hunk 1 above)
Comment (r-ahmed2022) [reply css/style.css:317]: Done as advised

File: index.html [hunk 7]
```diff
@@ -110,6 +110,183 @@ <h1 class="name-text">Hey There. <br>
...
+                    <p class="card-B-para">
+                      A daily selection of privately personalized <br> 
+                      reads;no accounts or sign-ups required.<br>
+                       has been the industry's standard
+                     </p>
+                     <div class="card-B-btns">
+                     <button type="button" class="btns-3">html</button>
+                     <button type="button" class="btns-3">bootstrap</button>
+                     <button type="button" class="btns-3">Ruby</button>
+                     </div>
```
[the remark this answers is not in the document]
Comment (r-ahmed2022) [reply index.html:192]: done as advised

**Comment by r-ahmed2022** [comment]
Implemented all required and optional changes as advised

File: index.html [hunk 8]
```diff
@@ -110,6 +110,195 @@ <h1 class="name-text">Hey There. <br>
...
+                    reads;no accounts or sign-ups required.<br>
+                     has been the industry's standard
+                   </p>
+                   <div class="card-B-btns">
+                    <ul>
+                      <li><a class="btns-3" href="#">html</a></li>
+                      <li><a class="btns-3" href="#">bootstrap</a></li>
+                      <li><a class="btns-3" href="#">Ruby</a></li>
+                      </ul>
+                   </div>
```
Comment (phelian23) [inline index.html:160]: - [ ] [Optional] There is no need to wrap a `<ul>` tag in a `<div>` container, as it is a block-level element.

**Review by phelian23 — APPROVED** [review]
Hi @r-ahmed2022, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**